pegasus/sites.v3/code.org/views/stats_homepage.haml

```diff
@@ -0,0 +1,21 @@
+- student_accounts_percent = "20%"
+- female_students = "9M"
+- teacher_users = "600K"
+- world_students_hoc_percent = "10%"
+- states_policy_change = "20"
+
```

Comment:
Agreed, if values are stored in the DB, the values should be stored as integers or floats. Let me know if you want help with the writing to or reading from `pegasus.properties`.

How often do we expect these values to change? What do we expect to trigger a change in values? Depending on the answers, it may make sense to define them as constants in code (perhaps scoped somewhere for reuse) rather than in the DB.